### PR TITLE
Integrate free start compute.

### PR DESCRIPTION
**Pull request by mariacarmina**
Fixes # .

Changes proposed in this PR:

- 
- 
In order to be tested locally, you need the branch for free start compute for ocean-node.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -88,7 +88,7 @@
     "@libp2p/tcp": "^10.0.7",
     "@libp2p/websockets": "^9.0.6",
     "@multiformats/multiaddr": "^12.3.1",
-    "@oceanprotocol/lib": "^3.3.3",
+    "@oceanprotocol/lib": "^4.0.0-next.1",
```
Comment (jamiehewitt15) [inline package.json:91]: We shouldn't be using this `4.0.0-next.1`, Alex said he was going to remove it... We should be using npm link to test this.

File: src/helpers/freeCompute.ts [hunk 2]
```diff
@@ -0,0 +1,350 @@
...
+  )
+
+  if (computeJobs && computeJobs[0]) {
+    const { jobId, agreementId } = computeJobs[0]
+    console.log('Compute started.  JobID: ' + jobId)
+    console.log('Agreement ID: ' + agreementId)
+  } else {
+    console.log('Error while starting the compute job: ', computeJobs)
+  }
+}
```
[the remark this answers is not in the document]
Comment (mariacarmina) [reply src/helpers/freeCompute.ts:350]: You have a new parameter which decides if you use free start compute or not: chainComputeEnv.free ? true : false

[the remark this answers is not in the document]
Comment (mariacarmina) [reply src/helpers/freeCompute.ts]: Had a previous commit with fetch and directCommand https://github.com/oceanprotocol/vscode-extension/pull/17/commits/dbdc3954a704686f5d82c80b6e732a7ce7bed024. I rolled back the changes

**Comment by jamiehewitt15** [comment]
Starting a free compute job works now:

image

**Comment by jamiehewitt15** [comment]
I'll work on https://github.com/oceanprotocol/vscode-extension/issues/23 and https://github.com/oceanprotocol/vscode-extension/issues/22 in separate PRs

**Review by jamiehewitt15 — APPROVED** [review]

**Pull request merged**